plugins/@grouparoo/bigquery/__tests__/integration/cli-generate-from.ts

```diff
@@ -15,8 +15,8 @@ const appOptions: SimpleAppOptions = loadAppOptions(newNock);
 const usersTableName = "records";
 
 import { api } from "actionhero";
-import { Generate } from "@grouparoo/core/src/bin/generate";
-import { Apply } from "@grouparoo/core/src/bin/apply";
+import { Generate } from "@grouparoo/core/dist/bin/generate";
+import { Apply } from "@grouparoo/core/dist/bin/apply";
```

Comment:
Why is this pulling from dist now?

Reply:
Parts of this suite already pull from dist (within the CLI command itself), and there were a few types not quite matching up.  This way the whole test now uses dist.  Also, these suites are going away soon when we remove the generate command!